subliminal/cli.py

```diff
@@ -271,11 +271,12 @@ def cache(ctx, clear_subliminal):
 @click.option('-m', '--min-score', type=click.IntRange(0, 100), default=0, help='Minimum score for a subtitle '
               'to be downloaded (0 to 100).')
 @click.option('-w', '--max-workers', type=click.IntRange(1, 50), default=None, help='Maximum number of threads to use.')
+@click.option('--scan-archives', is_flag=True, default=False, help='Scan archives for videos (Supported formats: Rar)')
```

Comment:
Letter `z` is usually associated with compression even if that's for `zip`, I think it might be appropriate as the flag is for archives in general (we could support other formats in the future). Otherwise the uppercase `A` is OK too.